### PR TITLE
Added personal hadron v3 layout called sebaslayout

### PR DESCRIPTION
**Pull request by Rionlion100**
## Description

Added personal keymap to /hadron/ver3/keymap
Features: right align numpad and 3u space 
## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/Layout/Userspace (**addition** or update)
- [ ] Documentation

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/hadron/ver3/keymaps/sebaslayout/keymap.c [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#include QMK_KEYBOARD_H
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	KEYMAP(
```
Comment (mechmerlin) [inline keyboards/hadron/ver3/keymaps/sebaslayout/keymap.c]: We are trying to move away from using the `KEYMAP` macro name. Please change these to `LAYOUT`.

File: keyboards/hadron/ver3/keymaps/sebaslayout/keymap.c [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+void matrix_scan_user(void) {
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+	return true;
+}
+
+void led_set_user(uint8_t usb_led) {
+
+	if (usb_led & (1 << USB_LED_NUM_LOCK)) {
```
Comment (mechmerlin) [inline keyboards/hadron/ver3/keymaps/sebaslayout/keymap.c]: Lines 41-69 can be removed.

Comment (Rionlion100) [reply keyboards/hadron/ver3/keymaps/sebaslayout/keymap.c]: Thanks will do

Comment (Rionlion100) [reply keyboards/hadron/ver3/keymaps/sebaslayout/keymap.c]: Done and Done

File: keyboards/hadron/ver3/keymaps/sebaslayout/keymap.c [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	LAYOUT(
+		KC_TRNS, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_F11, KC_F12, KC_TRNS, KC_HOME, KC_INS, KC_PGUP, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_END, KC_DEL, KC_PGDN, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_VOLU, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_VOLD, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS),
+
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {
```
Comment (drashna) [inline keyboards/hadron/ver3/keymaps/sebaslayout/keymap.c]: Could you remove this function completely?  

It's depreciated, and should be removed.

Comment (Rionlion100) [reply keyboards/hadron/ver3/keymaps/sebaslayout/keymap.c]: Think i fixed it?

**Pull request closed without merging**